tikv/pd
# Set the PD role to Unknown when pd_tso_role value is null

Signed-off-by: Hua Lu <hua.lu@pingcap.com>



### What problem does this PR solve?


Issue Number: Close #5047 

### What is changed and how does it work?
Change the `valueMaps` definition for `PD role` panel to Unknown when `pd_tso_role` value is null, 

### Check List



Tests



- Manual test (add detailed scripts or steps below)
  - Apply the changes to grafana
  - Check the role value is Unknown when the `pd_tso_role` is null or during the metric loading time

Code changes


Side effects


Related changes


### Release note
None
